Backend/api/user/router.js

```diff
@@ -5,31 +5,48 @@ const {
   checkAdminAuth,
 } = require("../middleware/auth-validation");
 const {
+  validationResults,
   isUserValid,
-  valdiationResults,
+  isWorkoutValid,
 } = require("../middleware/body-validation");
 
 const router = require("express").Router();
 
-router.get("/:id?", checkUserAuth, userController.getUser);
+router.get("/get/:_id?", checkUserAuth, userController.getUser);
```

Comment:
How about `/users/:_id`? Of course the workout routes need to be updated as well by prepending `/users`.
Don't forget to update the other CRUD user routes.

Reply:
I left it as it was before because I think is better to have the routes for users `{{base_url}}/api/user/:_userId` and for workouts `{{base_url}}/api/user/:_userId/workouts` instead of `{{base_url}}/api/user/users/:_userId` and `{{base_url}}/api/user/users/:_userId/workouts`